content/docs/started/getting-started-minikube.md

```diff
@@ -233,12 +233,13 @@ The following steps will deploy Kubeflow components and start them on the Miniku
 The above installation may take a few minutes. At the end of the installation you should see:
 ```
 Access Kubeflow dashboard at http://localhost:8080/
-Access JupyterHub at http://localhost:8080/hub/
+Access Jupyter notebooks at http://localhost:8080/notebooks/
 ```
 
 ### Where to go next
-Now you can access the Kubeflow dashboard at http://localhost:8080/ and JupyterHub at http://localhost:8080/hub/.
-For JupyterHub, you'll be landing on a login page.
+Now you can access the Kubeflow dashboard at http://localhost:8080/ and Jupyter
+notebooks at http://localhost:8080/notebooks/.
```

Comment:
Same suggestion as line 12 of this doc

Reply:
I'm going to stick with "Jupyter notebooks" for simplicity across the docs, as I think that will be clearer to our readers. It may come across as a little confusing to sometimes use "Notebook" and other times "notebook", though I do take your point about correctness here.